.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.4.0
```

Comment:
My bad! Sorry. I built this on OSX and 2.4.1 is a nightmare hence the local change.